app/policies/supported_permission_policy.rb

```diff
@@ -0,0 +1,16 @@
+class SupportedPermissionPolicy < BasePolicy
+  class Scope < ::BasePolicy::Scope
+    def resolve
+      if current_user.superadmin?
+        scope.all
+      elsif current_user.admin?
+        scope.all
+      elsif current_user.organisation_admin?
+        app_ids = Pundit.policy_scope(current_user, :manage_supported_permissions_for_application).pluck(:id)
```

Comment:
Don't `SupportPermissionsPolicy` and `ManageSupportedPermissionsForApplicationPolicy` do essentially the same thing? 
What's the added benefit of keeping the logic for `ManageSupportedPermissionsForApplicationPolicy` separated from `SupportedPermissionPolicy`?